.github/workflows/build.yml

```diff
@@ -0,0 +1,54 @@
+name: Build and Deploy to Dev
+
+on:
+  push:
+    tags:
+      - 'dev'
+
+env:
+  ACCOUNT_ID: ${{ secrets.ACCOUNT_ID }}
+  S3_BUCKET: ${{ secrets.S3_BUCKET }}
+  PROJECT: enhanced-travel-screening
+  DEPLOY_ENV: enhanced-travel-screening-dev
+  REGION: ca-central-1
+
+jobs:
+  test:
```

Comment:
I'm not sure if this terminology should really be **_test_** if we are just running **_npm audit_**. Because we have actual tests already on /server and it might be confusing. Unless we are also planning to add them here afterwards.

Reply:
Totally agree. This was a placeholder for actually running the tests. Just never implemented.